index.html

```diff
@@ -42,5 +43,224 @@ <h2 class="Supportingtext Title2">I'm a Software Developer</h2>
       </ul>
     </div>
   </section>
+
+<!-- Section: Recent Works -->
+<section class="RecentWorks">
+
+  <div class="RecentWorksSection">
+    <h3 class="Title_MyRecentWorks">My recent Works</h3>
+  </div>
+  
+  <!-- ************ grid used to display diferent project cards ************ -->
+
+  <div class="grid_works">   
+
+    <!-- ************* Main card ************* -->
+    
+    <div class="main_card_cnt">
+      <div class="cnt_img_main_card">
+        <img class="main_card_cnt_img" src="/images/card1_img.png" alt="Project image">
```

Comment:
- **[OPTIONAL]** A simple dot could do a big difference in our code, please correct the path of the image by adding a dot at the beginning to avoid broken images. 👇🏽 

| Without dot | With dot |
|---|---|
| ![image](https://user-images.githubusercontent.com/58642949/228700087-0e3d4e85-4e48-492e-9c5f-c24bbe39d00a.png) | ![image](https://user-images.githubusercontent.com/58642949/228700196-5bb7a28c-b05e-4601-b0a4-c3f7525462a8.png) |

Reply:
Thank You! Done.